manage.py

```diff
@@ -26,6 +26,21 @@ def make_shell_context():
 manager.add_command('db', MigrateCommand)
 
 
+def create_default_admin(first_name, last_name, email, password):
+    if (User.query.filter_by(email=email).first() is not None):
```

Comment:
It's actually not a pep8 thing, just a pycharm thing. But you're right, they are redundant.